recipes/pygnome/meta.yaml

```diff
@@ -0,0 +1,85 @@
+{% set name = "PyGnome" %}
+{% set version = "1.1.11" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/NOAA-ORR-ERD/PyGnome/archive/v{{ version }}.tar.gz
+  sha256: f4263ea7e373414ea66ca0f630b705b07e0131258b2fafad03d369b0f5ef0ee6
+
+build:
+  skip: true  # [linux]
+  number: 0
+  script: cd py_gnome && {{ PYTHON }} -m pip install ./ -vv
+
+requirements:
+  build:
+    - cython
+    - numpy
+    - netCDF4
+    - scikit-build-core
+    - clang 15.*
+    - cmake
+  host:
+    - python >=3.7
+    - pip
+    - numpy
+    - scikit-build-core
+  run:
+    - netCDF4
+    - openssl
+    - libzlib
+    - bzip2
+
+test:
+  imports:
+    - gnome
+  requires:
```

Comment:
I don't think you need all the requirements here -- just the ones that are extra for the tests.

Reply:
I was failing the import test without the presence of those python packages.